apps/pollbook/frontend/src/address_input_group.tsx

```diff
@@ -42,13 +42,66 @@ function findDetailsFromStreetAddress(
   };
 }
 
-export function splitStreetNumberAndSuffix(input: string): {
+export function splitStreetNumberDetails(input: string): {
   streetNumber: string;
   streetSuffix: string;
+  houseFractionNumber: string;
+  useHouseFractionSeparator: boolean;
 } {
-  const match = input.match(/(\d+)(.*)/);
-  const [, streetNumber = '', streetSuffix = ''] = match ?? [];
-  return { streetNumber, streetSuffix };
+  // First, try to match the complete valid pattern
+  // This regex handles multiple cases:
+  // - Street number only: "16"
+  // - Street number + suffix: "16B" or "16 B"
+  // - Street number + fraction: "16 1/2" or "161/2"
+  // - Street number + incomplete fraction: "16 1/"
```

Comment:
Why do we want to support incomplete fractions?

Reply:
because the input has to keep workign as you are in the process of typing it out